docs/design/features/host-error-codes.md

```diff
@@ -1,111 +1,51 @@
 # Hosting layer error and exit codes
 This document lists all the values returned as special exit codes when running `dotnet.exe` or `apphost` or returned by the hosting APIs (`hostfxr`, `hostpolicy` and `nethost`).
+Those exit codes are also specified in signed 32-bit form (what you would see if you run `echo %ERRORLEVEL%` on Winodws) and in 8-bit form (what you would see if you run `echo $?` on Linux).
 Note that the exit code returned by running an application via `dotnet.exe` or `apphost` can either be one of the below values or the exit code of the managed application itself.
 
-
 ### Success error/exit codes
-
-* `Success` (`0`) - Operation was successful.
-
-* `Success_HostAlreadyInitialized` (`0x00000001`) - Initialization was successful, but another host context is already initialized, so the returned context is "secondary". The requested context was otherwise fully compatible with the already initialized context. This is returned by `hostfxr_initialize_for_runtime_config` if it's called when the host is already initialized in the process. Comes from `corehost_initialize` in `hostpolicy`.
-
-* `Success_DifferentRuntimeProperties` (`0x00000002`) - Initialization was successful, but another host context is already initialized and the requested context specified some runtime properties which are not the same (either in value or in presence) to the already initialized context. This is returned by `hostfxr_initialize_for_runtime_config` if it's called when the host is already initialized in the process. Comes from `corehost_initialize` in `hostpolicy`.
+| Name                                  | Value        | Value (signed, 32-bit) | Value (8-bit) | Description |
+| ------------------------------------- | ------------ | ---------------------- | ------------- | ----------- |
+| `Success`                             | `0x00000000` | `-4294967296`          | `0`           | Operation was successful. |
+| `Success_HostAlreadyInitialized`      | `0x00000001` | `-4294967295`          | `1`           | Initialization was successful, but another host context is already initialized, so the returned context is "secondary". The requested context was otherwise fully compatible with the already initialized context. This is returned by `hostfxr_initialize_for_runtime_config` if it's called when the host is already initialized in the process. Comes from `corehost_initialize` in `hostpolicy`. |
```

Comment:
```suggestion
| `Success_HostAlreadyInitialized`      | `0x00000001` | `1`          | `1`           | Initialization was successful, but another host context is already initialized, so the returned context is "secondary". The requested context was otherwise fully compatible with the already initialized context. This is returned by `hostfxr_initialize_for_runtime_config` if it's called when the host is already initialized in the process. Comes from `corehost_initialize` in `hostpolicy`. |
```